my-app/src/Components/Navbar/Navbar.tsx

```diff
@@ -10,14 +10,15 @@ import InputOutput from '../InputOutput/InputOutput';
 
 const Navbar = () => {
     const {isActive} = useSelector((state) => state.active);
+    const toggleSideBar = () => dispatch(toggleActive());
     const dispatch = useDispatch();
     return (
         <div className={!isActive ? style.navbar : `${style.navbar} ${style.active}`}>
             <div className = {style.menuBox}>
-                <NavLink to = '/user' className = {style.userPoint} onClick={() => dispatch(toggleActive())}><User userName= {'Artem Malkin'}/></NavLink>
-                <NavLink to = '/' className = {style.menuPoint} onClick={() => dispatch(toggleActive())}>Home</NavLink>
+                <NavLink to = '/user' className = {style.userPoint} onClick = {toggleSideBar}><User userName= {'Artem Malkin'}/></NavLink>
```

Comment:
please remove unnecessary spaces everywhere. so instead of onClick = {toggleSideBar} u should write onClick={toggleSideBar}